app/controllers/api/deploys_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::DeploysController < Api::BaseController
+  include CurrentProject
+
+  def index
+    scope = current_project.try(:deploys) || Deploy
```

Comment:
This is copy pasta from deploys_controller.rb I need to re-extract to serializers.